app/src/main/res/layout/fragment_receive.xml

```diff
@@ -98,22 +96,30 @@
 
     </LinearLayout>
 
-    <Button
-        android:id="@+id/bGenerate"
-        style="@style/MoneroButton"
+    <FrameLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginBottom="4dp"
-        android:layout_marginTop="4dp"
-        android:enabled="false"
-        android:minHeight="36dp"
-        android:text="@string/receive_generate_hint" />
-
-    <ImageView
-        android:id="@+id/qrCode"
-        android:layout_width="fill_parent"
-        android:layout_height="fill_parent"
-        android:adjustViewBounds="true" />
+        android:layout_height="match_parent">
+
+        <TextView
+            android:id="@+id/tvQrCode"
+            style="@style/MoneroFab"
```

Comment:
imo that's an unlucky naming, I somehow think of a floating action button(fab) with this style.

Reply:
it **is** the FAB text style - maybe it should be renamed to MoneroFabText